.rubocop.yml

```diff
@@ -1,3 +1,9 @@
+inherit_from: .rubocop_todo.yml
+
+# Exclude legacy violations. The todo list should only get SMALLER
+# Once it is empty this can be removed!
+# inherit_from: .rubocop_todo.yml
+
 AllCops:
   TargetRubyVersion: 2.4.2
```

Comment:
We're on ruby 2.5.1 ;)

Reply:
Well... that was there before. But I updated it :)